uint32       current_key        #当前使用和显示载车的id
uint32       gps_week           #GPS周，时间零点为1980年1月6日零点，每1024周为一个循环周期
uint64       gps_millisecond    #GPS毫秒，以周的零点开始的毫秒数
uint8        reserve1           #预留
uint8        reserve2
uint8        reserve3
uint8        reserve4
uint8        reserve5
uint8        reserve6
uint8        reserve7
uint8        reserve8
